packages/workspace/src/command-line/workspace-generators.ts

```diff
@@ -65,7 +65,14 @@ function compileTools() {
 }
 
 function getToolsOutDir() {
-  return path.resolve(toolsDir, toolsTsConfig().compilerOptions.outDir);
+  const outDir = toolsTsConfig().compilerOptions.outDir;
+
+  if (!outDir) {
+    logger.error('tsconfig.tools.json must specify an outDir');
```

Comment:
Can you update this to use this?
```suggestion
logger.error(`${toolsTsConfigPath} must specify an outDir`);
```

Reply:
Good call. I've added the file path to the error message.